hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -13,8 +13,10 @@
 from tempfile import NamedTemporaryFile, TemporaryDirectory
 from typing import List
 
+import numpy as np
 from hyp3_metadata import create_metadata_file_set
```

Comment:
This function and all of it's calls will need to be renamed to `create_metadata_file_set_rtc`